ores/wsgi/static/js/scorer.js

```diff
@@ -79,14 +79,14 @@ function getResults() {
 	});
 	models_url = models_url.slice(0, -1);
 	var url = "/scores/" + $('#wikiDropDownInput').attr('value') + "/?models=" + models_url + "&revids=" + revs;
-	$.get(url, function (data) {
+	$.get(url, datatype='jsonp').always(function(data) {
```

Comment:
Why not e.g. `/* datatype= */ 'jsonp'` to avoid creating unused variable? :wink: